### DISCUSSION TITLE
How to centre a resized image?

### DISCUSSION THREAD

**Discussion by joshuaknipe** [jpanther/congo#916] [Q&A]
Hi there! Thanks for everyone's' work on this awesome project! I installed Congo via the Hugo Modules method so unfortunately cannot use the tailwind jit complier for this solution: https://github.com/jpanther/congo/discussions/323 
Is there any other way I can centre a resized image? I have created a custom.css file in assets/css with the following content:

.center-image {
    display: block;
    margin-left: auto;
    margin-right: auto;
}

My actual post then has the following shortcode:

{{< figure
    src="images/calendar.png"
    alt="Calendar plugin example"
    caption="Calendar plugin"
    default=true
    width="50%"
    class="centre-image"
    >}}

but the image is still left aligned 

Thank you!

**Comment by GorkemGuray** [comment]
check this
https://github.com/jpanther/congo/discussions/771#discussioncomment-8070692
